packages/amplication-server/src/core/entity/dto/EntityCreateInput.ts

```diff
@@ -5,6 +5,13 @@ import { WhereParentIdInput } from "../../../dto";
   isAbstract: true,
 })
 export class EntityCreateInput {
+  @Field(() => String, {
+    nullable: true,
+    description:
+      "allow creating the id for the entity when using import prisma schema because we need it for the relation",
+  })
+  id?: string;
+
```

Comment:
@abrl91 is this still needed?

Reply:
yes, we need it because we use the id that we generate here:
![image](https://github.com/amplication/amplication/assets/39680385/5e0bfecb-7024-4c0f-b1df-c240cf78fec3)